tools/apidiff/delta/delta.go

```diff
@@ -0,0 +1,345 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package delta
```

Comment:
Package `delta` has been refactored previously, now it is in `tools/internal/delta`, please use that instead

Reply:
fixed